ui/src/api/orders/order.js

```diff
@@ -0,0 +1,113 @@
+import { http, apiConfig } from '../base';
```

Comment:
Can I propose to have the folder structure:
`/api/acquisitions/orders.js`
? WDYT?

Reply:
The only issue is where do we put the serializer and tests? Perhaps `/api/acquisitions/orders/order.js` to follow the same structure as the other apis?